plugins/sndio/data/locale/ru-RU.ini

```diff
@@ -0,0 +1,6 @@
+Channels="Количество каналов"
```

Comment:
Dont include non-english locales in PRs. These should be contributed via http://crowdin.com/project/obs-studio

Reply:
done